utils/avro-components-utils/src/main/scala/pl/touk/nussknacker/engine/avro/typed/AvroSchemaTypeDefinitionExtractor.scala

```diff
@@ -81,26 +81,7 @@ class AvroSchemaTypeDefinitionExtractor(skipOptionalFields: Boolean) {
       case Schema.Type.BOOLEAN =>
         Typed[Boolean]
       case Schema.Type.NULL =>
-        Typed.empty
+        TypedNull
```

Comment:
I don't think that it is a good idea. This class show what type will be returned in source and it is the only purpose of it. (maybe we should change the name of class to avoid confusions?). For simple schema null | other type we will present union type which 1) is verbose and it doesn't help user 2) doesn't work correctly for some edge cases - don't remember which once.

Reply:
My bad, It's good - we filtering out null from unions